apollo-federation/Cargo.toml

```diff
@@ -36,6 +36,7 @@ strum_macros = "0.26.0"
 thiserror = "1.0"
 url = "2"
 tracing = "0.1.40"
+ron = { version = "0.8.1", optional = true }
```

Comment:
should this be dev dependency instead?

Reply:
No. This is not used for tests. The visualizer uses this for snapshots. If you want to get snapshot logs while testings, you can just enable that feature when you run `cargo test`.